src/iohumdrum.cpp

```diff
@@ -671,22 +756,22 @@ void HumdrumInput::getRespPeople(vector<vector<string> > &respPeople, vector<Hum
 {
 
     // precalculate a reference map here to make more O(N) rather than O(N^2)
-    addPerson(respPeople, references, "COM", "Composer"); // cmp
+    addPerson(respPeople, references, "COM", "composer"); // cmp
```

Comment:
This is approximately the list I was using for roles:
https://www.loc.gov/marc/relators/relaterm.html

Here is a related discussion from @pe-ro to me in March 2013:

Moving on, just for fun, you can combine `<composer>` and `<persName>` like so --
```xml
<composer type="attributed">
  <persName authority="LCNAF" authURI="http://lccn.loc.gov/n80005115">Josquin</persName>
</composer>
```
This asserts that Josquin is the attributed composer and that the authorized form of "Josquin" (in the Library of Congress Name Authority File) is stored at http://... (which is "Josquin, des Prez, d. 1521", by the way).

 
Currently, there's no place to record the authorized form of the name directly in MEI.  The quick and dirty way to do this is to use an attribute (something like `@reg`) to store the authorized name, but that's "overloading" an attribute.  On the other hand, providing for multiple options within `<composer>` complicates the markup --

```xml
<composer type="attributed">
  <choice>
    <orig>Josquin</orig>
    <reg>
      <persName authority="LCNAF" authURI="http://lccn.loc.gov/n80005115"/>
    </reg>
  </choice>
</composer>
```

but this isn't allowed yet.  I'm not sure `<choice>` is a good idea for use in the header.